File: C_Programming/Basic C Programming/odd_even.c
#include <stdio.h>

int main () {
    
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);
    
    if (n % 2 == 0) {
        printf("\n%d is an Even number", n);
    }
    else 
    printf("\n%d is a Odd number", n);
    
    return 0;
    
}

File: PHP/Odd_Even.php
<?php
// // Function to check if a number is odd or even
// function checkOddEven($num) {
//     if ($num % 2 == 0) {
//         echo "$num is an Even number.";
//     } else {
//         echo "$num is an Odd number.";
//     }
// }

// // Example usage
// $number = 2; // Change this number to test
// checkOddEven($number);


$x = 4;
($x % 2 == 0) ? print "$x is an Even number." : print "$x is an Odd number."; // print is slower than echo
echo "<br>";
echo ($x % 2 == 0) ? "$x is an Even number." : "$x is an Odd number."; // echo is faster than print
?>